fairscale/optim/oss.py

```diff
@@ -44,12 +45,25 @@ class OSS(Optimizer):
             optimizer to shard (default: SGD)
         group (group):
             torch.distributed group (default: group.WORLD)
+        buffer_size (int, optional): number of elements to buffer before
```

Comment:
What does optional mean in this context? The parameter does not look like an optional.

Reply:
I meant to write that people are free to pass it in or not, there's a default provided